### PR TITLE
[UXE-2422] refactor: l2 cache to tiered cache labels, variables and functions names

### PR DESCRIPTION
**Pull request by peterpaulo-azion**
### What is the new behavior introduced by this PR?

refactor - l2 cache to tiered cache

### Does this PR introduce breaking changes?
- [ ] No
- [x] Yes 

Rename Product and project structure to new name of l2 cache. Now any l2 caching reference will be tiered cache.

### Does it have a link on Figma?

<hr />

#### Make sure your pull request fits the checklist below (when applicable):

- [x] The issue title follows the format: [ISSUE_CODE] TYPE: TITLE
- [x] Commits are tagged with the right word (fix, feat, test, etc)
- [x] User inputs are sanitized/protected against malicious attacks
- [ ] Tags are added to the PR

#### These changes were tested on the following browsers:
- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Safari

### CODE REVIEW

**Review by Erick374 — DISMISSED** [review]

File: src/services/metrics-services/constants/dashboards-texts.js [hunk 1]
```diff
@@ -162,13 +162,13 @@ const DASHBOARDS_TEXTS = {
       }
     }
   },
-  l2_caching: {
+  tiered_caching: {
```
Comment (william-tome) [inline src/services/metrics-services/constants/dashboards-texts.js:165]: Confere o nome desse produto aqui e nos outros pontos. Acho que é "tiered cache" ao invés de "tiered caching"

File: src/services/metrics-services/constants/dashboards-texts.js [hunk 2]
```diff
@@ -162,13 +162,13 @@ const DASHBOARDS_TEXTS = {
...
     caching_offload: {
-      l2_caching: {
+      tiered_caching: {
         description:
           'Sum of data transferred through Tiered Cache. Displays the data in bytes divided by Data Transferred Total, Data Transferred In, and Data Transferred Out.'
       },
-      l2_offload: {
+      tiered_offload: {
```
Comment (william-tome) [inline src/services/metrics-services/constants/dashboards-texts.js:171]: Esse nome está correto?

File: src/services/metrics-services/constants/dashboards-texts.js (hunk 1 above)
Comment (peterpaulo-azion) [reply src/services/metrics-services/constants/dashboards-texts.js]: Feito

File: src/services/metrics-services/constants/dashboards-texts.js (hunk 2 above)
Comment (peterpaulo-azion) [reply src/services/metrics-services/constants/dashboards-texts.js]: Feito

**Review by matheus-sunderhus — APPROVED** [review]

**Review by william-tome — APPROVED** [review]

**Pull request merged**